packages/tfchain_client/lib/src/client.dart

```diff
@@ -0,0 +1,367 @@
+    return _policies!;
+  }
+
+  QueryBridge get bridge {
+    if (_bridge == null) _bridge = QueryBridge(this);
+    return _bridge!;
+  }
+
+  QueryTFTPrice get price {
+    if (_price == null) _price = QueryTFTPrice(this);
+    return _price!;
+  }
+
+  Dao.QueryDao get dao {
+    if (_dao == null) _dao = Dao.QueryDao(this);
+    return _dao!;
+  }
+
+  void _checkInputs() {
+    if (url.isEmpty) {
+      throw FormatException("URL should be provided");
+    }
+  }
+
+  Future<void> connect() async {
+    _checkInputs();
+    provider = Provider.fromUri(Uri.parse(url));
+    api = polkadot.Dev(provider!);
+  }
+
+  Future<void> disconnect() async {
+    if (provider != null && provider!.isConnected()) {
+      await api.disconnect();
+    }
+  }
+}
+
+class Client extends QueryClient {
+  final String mnemonicOrSecretSeed;
+  late String address;
+  final String _keypairType;
+  late final KeyPair? keypair;
+  KVStore? _kvStore;
+  Signer.KPType? _type;
+  TermsAndConditions? _termsAndConditions;
+  static const List<String> _SUPPORTED_KEYPAIR_TYPES = ["sr25519", "ed25519"];
+
+  Client(String url, this.mnemonicOrSecretSeed, this._keypairType)
+      : super(url) {}
+
+  @override
+  Twins get twins {
+    if (_twins == null) _twins = Twins(this);
+    return _twins as Twins;
+  }
+
+  @override
+  Contracts get contracts {
+    if (_contracts == null) _contracts = Contracts(this);
+    return _contracts as Contracts;
+  }
+
+  @override
+  balance.Balances get balances {
+    if (_balances == null) _balances = balance.Balances(this);
+    return _balances as balance.Balances;
+  }
+
+  @override
+  Farms get farms {
+    if (_farms == null) _farms = Farms(this);
+    return _farms as Farms;
+  }
+
+  @override
+  Nodes get nodes {
+    if (_nodes == null) _nodes = Nodes(this);
+    return _nodes as Nodes;
+  }
+
+  @override
+  Bridge get bridge {
+    if (_bridge == null) _bridge = Bridge(this);
+    return _bridge as Bridge;
+  }
+
+  @override
+  Dao.Dao get dao {
+    if (_dao == null) _dao = Dao.Dao(this);
+    return _dao as Dao.Dao;
+  }
+
+  KVStore get kvStore {
+    if (_kvStore == null) _kvStore = KVStore(this);
+    return _kvStore as KVStore;
+  }
+
+  TermsAndConditions get termsAndConditions {
+    if (_termsAndConditions == null)
+      _termsAndConditions = TermsAndConditions(this);
+    return _termsAndConditions as TermsAndConditions;
+  }
+
+  @override
+  void _checkInputs() {
+    super._checkInputs();
+    if (!_SUPPORTED_KEYPAIR_TYPES.contains(_keypairType)) {
+      throw FormatException(
+          "Keypair type $_keypairType is not valid. It Should be either of : ${_SUPPORTED_KEYPAIR_TYPES}");
+    }
+
+    if (_keypairType == "sr25519") {
+      _type = Signer.KPType.sr25519;
+    } else {
+      _type = Signer.KPType.ed25519;
+    }
+
+    if (mnemonicOrSecretSeed.isEmpty) {
+      throw FormatException("Mnemonic or secret should be provided");
+    } else if (mnemonicOrSecretSeed == "//Alice") {
```

Comment:
I found that it can be any dev account like `//Alice` or `//Bob`, so let's check if it starts with `//` and that will be a dev account